server/populateIdDatabaseScript.js

```diff
@@ -28,20 +28,36 @@ const foodTypes = [
   'daal',
   'kale',
   'spinach',
-  'cheese'
+  'cheese',
+  'hummus',
+  'bean'
 ];
 
-
-var options = {
-  uri: `http://api.yummly.com/v1/api/recipes?_app_id=${config.APP_ID}&_app_key=${config.APP_KEY}&allowedDiet[]=392^Vegetarian&q=${foodParameter}&requirePictures=true&maxResult=${resultLimit}`,
-  json: true
-};
- 
-rp(options)
+foodTypes.forEach(foodType => {
+  rp({
```

Comment:
How is the DB being queried with this command (it just uses 'rp'; I don't see a specific method being invoked)?